monitoring/grafana/grr_grafanalib_dashboards/config.py

```diff
@@ -0,0 +1,5 @@
+from grr_grafanalib_dashboards import reusable_panels
+
+DATA_SOURCE = "grr-server"
+ACTIVE_PROCESSES_CONDITION = 1
+PANELS = [reusable_panels.number_of_active_processes_graph, reusable_panels.avg_cpu_usage_percentage]
```

Comment:
Would it make sense to just put this constant into reusable_panels module? What do we gain by having it "config"?

Reply:
I agree that this can go into `reusable_panels.py` straight away, but I think that the variables `DATA_SOURCE` and `ACTIVE_PROCESSES_CONDITION` are not related to the general panels whatsoever, and this is why I kept them in different files.

Reply:
Totally agreed. Sorry for not being clear. My comment was related to PANELS only.